doc/ref_impl/cntt-ri/chapters/chapter06.md

```diff
@@ -61,9 +63,385 @@ Thanks to the descriptor file, the NFVi infrastructure deployment could be compl
+| SKU | String | Yes | SKU of server |
+| MODEL | String | Yes | server service type defined by provider/user, same definition as in above table, example: NC1 or NC2  |
+| NETWORK_PLANE | String | Yes | Network plane for each nic, Manage or Storage or Service  |
+| PORT | String | Yes | Ports number for example: 1_1  |
+| PORT_BDF | String | Yes | Port BDF value for above port  |
+
+<p align="center"><b>Table 6-3-4-3:</b> Port BDF Information.</p>
+
+<a name="6.3.5"></a>
+### 6.3.5 Network Device information
+This table describes each network device, it can be used for network configuration and verification.
+
+| Field # | type | mandatory | Instruction |
+|----|--------------------|----------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| NAME | String | Yes | Name of network device |
+| VENDOR | String | Yes | Vendor name for network device  |
+| SKU | String | Yes | SKU |
+| MODEL | String | Yes | Type of switch, like TOR or EOR |
+| SN | String | Yes | Serial number |
+| HW_RES_POOL | String | Yes | Resource pool name for hardware |
+| RACK | List | Yes | rack number where switch is placed |
+| POS | List | Yes | position in rack |
+| BMC_IP | List | Yes |  |
+| BMC_GATEWAY | List | Yes |  |
+| BMC_MASK | List | Yes |  |
+| BMC_USR | List | Yes | BMC login user |
+| BMC_PWD | List | Yes | password for BMC login user |
+| ENABLE_PWD | List | Yes | Enable password |
+| GROUP_NAME | List | Yes | Manage or storage or service |
+| HW_REGION | List | Yes | Hardware region |
+| MODULE_NAME | List | Yes | Hardware module which is devided by location, like area A module 1 |
+
+<p align="center"><b>Table 6-3-5:</b> Network device information.</p>
+
+
+<a name="6.3.6"></a>
+### 6.3.6 Port mapping information
+Wiremap defines the port mapping between server/switch and switch for each line, 
+we will need this information to trace the connected server and port, so we can extrapolate the required network configuration for the port.
+
+| Field # | type | mandatory | Instruction |
+|----|--------------------|----------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| NAME | String | Yes | Name of network device |
+| LOCAL_RACK | String | Yes | the rack info for local device   |
+| LOCAL_NAME | String | Yes | local device name |
+| LOCAL_TYPE | String | Yes | Local device type, switch or server |
+| LOCAL_PORT | String | Yes | connected port in local device |
+| REMOTE_RACK | String | Yes |  |
+| REMOTE_NAME | List | Yes | connected remote device name |
+| REMOTE_TYPE | List | Yes | remote device type |
+| REMOTE_PORT | List | Yes | connected port in remote device  |
```

Comment:
Taking below lines as example, we use the "LOCAL_TYPE" and "REMOTE_TYPE" to define whether it's connected to a remote switch or server. 
when describe port for remote servers, we use port number like 1_1, or 1_2, instead of PCIeslot number, the server NIC mapping is already defined in 6.3.4.2
And we also define "LINE_TYPE" to how each line is connected, like S-SRV-C_S-TOR means this line is connecting a service server in compute module to service TOR， and another example "ST-SRV-S_M-TOR" means storage server connecting to a manage TOR in storage module. And the line type can be customized defined.
LOCAL_NAME:"NFV-D-XNCD-06A-2203-0A01-S-TOR-01:
LOCAL_TYPE:"switch"
LOCAL_PORT:"10GE1/0/1"
REMOTE_NAME:"NFV-D-XNCD-06A-2203-0A01-M-SRV-01:
REMOTE_TYPE:"server"
REMOTE_PORT:"1_1"
LINE_TYPE:"S-SRV-C_S-TOR"